runtime/compiler/compile/J9SymbolReferenceTable.hpp

```diff
@@ -209,7 +209,7 @@ class SymbolReferenceTable : public OMR::SymbolReferenceTableConnector
     *  \param type
     *     The data type of the field.
     *  \param fieldOffset
-    *     The offset of the field.
+    *     The offset of the field from the first field.
```

Comment:
I would suggest changing this to read "The offset of the field from the beginning of the first field", just to prevent someone from thinking it might mean the offset from the end of the first field.

Reply:
Addressed in [3a216bd](https://github.com/eclipse-openj9/openj9/commit/3a216bd9dd80e89e99bdcdc61289366e65b5ed43)